physionet-django/user/templates/user/credential_application.html

```diff
@@ -16,6 +16,7 @@ <h1>Application for Credentialed Access</h1>
 
   <p>Please use the form below to apply for {{ SITE_NAME }} credentialing. In order to apply:</p>
   <ul>
+    <li>If you have an institutional/academic please add it in your settings (<a href="{% url 'edit_emails' %}">Emails settings</a>).</li>
```

Comment:
Looks like a word is missing here? "If you have an institutional/academic XXX please add it...".

Reply:
thanks @tompollard, added the email address to the text.